REPORT.md

```diff
@@ -0,0 +1,76 @@
+# Reporting capabilities within zAppBuild
+
+zAppBuild provides a set of reporting capabilities, which are part of the build framework itself to address some common demands of mainframe development teams.
+
+Developers are particularly interested if
+* a change of an element impacts other application components which are managed in a different repository.
+* their changes on an isolated feature branch potentially cause a conflict with some concurrent development done by others within the same repository.
+
+While during the analysis phase of a new task, the developer leverages application dependency understanding tools such as IBM Application Discovery, it is beneficial to enable also the build framework to automatically generate reports to answer the above questions.
+
+By default these reporting capabilities are turned off, while they don't belong to the core functionalities of a build framewoek.
```

Comment:
In the introduction it is stated, "zAppBuild provides a set of reporting capabilities, which are part of the build framework itself".  But hen there is the statement here, "while they don't belong to the core functionalities of a build framewoek" (BTW - framework is misspelled).  This left me a little confused.  Does the frame work support reporting or not.  I believe the answer is, Yes.

I think what you are trying to convey here is the functionality is turned off by default "since it is normally not part" of the standard build framework workflow.